scvi/module/base/_base_module.py

```diff
@@ -36,14 +38,35 @@ class LossRecorder:
 
     def __init__(
         self,
-        loss: Union[Dict[str, torch.Tensor], torch.Tensor],
+        loss: Union[
+            Dict[str, Union[torch.Tensor, jnp.ndarray]],
```

Comment:
I should really address https://github.com/scverse/scvi-tools/issues/1131

Reply:
yes later :)